### PR TITLE
Chaining improvements + new version release

### PR DESCRIPTION
**Pull request by santiycr**
This allows further commands to be run after the chain ends, while fixes the issue of TestComplete never being run when a test fails.
Also did some fixes in the examples. Couldn't update the api.html as dox change it's API. Letting that one to you guys.

Cheers

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "soda",
-  "description": "Selenium RC Node Adapter (with Saucelabs support)",
+  "description": "Selenium RC Node Adapter (with Sauce Labs support)",
   "keywords": ["selenium", "saucelabs", "testing", "test", "tests"],
-  "version": "0.2.4",
+  "version": "0.2.5",
```
Comment (tj) [inline package.json:5]: Y U BUMP

that's a no no haha

Comment (santiycr) [inline package.json:5]: Hah, didn't know how to add the fixes to the History, I guess that's your job. Feel free to revert ;)

Comment (tj) [inline package.json:5]: just bugging haha, but yeah typically best left to the author in case they want to fix/add or whatever

Comment (santiycr) [inline package.json:5]: Thanks, heh. Will keep it in mind :)

**Pull request merged**